### ISSUE TITLE
Clarify that callouts are optional

### ISSUE THREAD

**Issue by ErinBecker** [carpentries/instructor-training#1160]
This came up in my teaching demo this morning and has come up in several demos before. Trainees were surprised that callouts weren't a required part of the lesson. They were trying to teach all of the callouts, which is a timing problem, especially for some of the older SWC lessons where there are many callouts. It would be good to emphasize somewhere in training and/or documentation that callouts are optional extras that Instructors can include or skip depending on time, the questions that learners are asking, and interest in particular topics.

**Comment by brownsarahm** [comment]
This sounds like a helpful clarification to make.  We should discuss *where* to document this. 

Maybe it can be a callout in "Preparing to Teach" (`15-lesson-study`)? There's already a callout there for instructor notes.

**Comment by ChristinaLK** [comment]
I agree that putting a note in "preparing to teach" would be good. Could we add a "tour of the lessons" exercise or section where we go over the information contained in each lesson + how you would use it to prepare to teach?

**Comment by Denubis** [comment]
Perhaps we also want to highlight this during teaching demos? (Maybe as part of #1156)? I think that's the best place to repeat norms of "what makes a lesson" and "don't just read start to finish".

**Comment by brownsarahm** [comment]
Yes, this should also be added to either the [instructor notes](https://carpentries.github.io/instructor-training/guide/index.html#vi-teaching-demonstration-tips) and/or the [trainer guide](https://docs.carpentries.org/topic_folders/instructor_training/trainers_guide.html#running-a-teaching-demonstration) for trainers to remind people. 

I really like a tour of the lessons exercise. We could format it like the scavenger hunt activity on the lesson template from maintainer onboarding.

**Comment by ChristinaLK** [comment]
^^ @brownsarahm I'm pretty sure I was subconsciously channeling maintainer onboarding in my comment. :)

**Comment by Denubis** [comment]
So if we're thinking of this, it might be a useful replacement for the git segment (and then we figure out how to flip the spin-up-workshop-page/lesson) stuff so it's documented?

**Comment by ChristinaLK** [comment]
I think it could go in either "preparing to teach" or "the Carpentries". Time-wise, I like your proposal @Denubis
